src/components/CompareResults/ResultsMain.tsx

```diff
@@ -55,6 +80,19 @@ function ResultsMain() {
             <div className={styles.title}>Results</div>
             <div className={styles.content}>
               <SearchInput onChange={setSearchTerm} />
+
+              <FormControl sx={{ minWidth: 120 }}>
```

Comment:
this minWidth doesn't seem to work, it's still too small.

I'd suggest to remove it for now and deal with that in a future PR. I really think we need to use flex properly on this line (maybe with the MUI element `Grid` even though that's not the only way)

Reply:
I'll make a separate PR to fix the styles for this row.